client/src/pages/index.tsx

```diff
@@ -57,6 +57,10 @@ function IndexPage({
     allChallengeNode: { nodes: challengeNodes }
   }
 }: Props): JSX.Element {
+  useEffect(() => {
+    sessionStorage.setItem('canOpenModal', 'true');
+  }, []);
```

Comment:
We should not be setting this in production at all.

Reply:
> We should not be setting this in production at all.

what do you think we should do instead ?

Reply:
Refer to teh example in the issue?

```
if (process.env.environment == 'development' {
	const activate = localStorage.getItem("activateMyFeature");
	if (activate) {
		//... do not submit!!
	}
}
```

Reply:
> Refer to teh example in the issue?
> 
> ```
> if (process.env.environment == 'development' {
> 	const activate = localStorage.getItem("activateMyFeature");
> 	if (activate) {
> 		//... do not submit!!
> 	}
> }
> ```

Should I should create a storage item only if the environment is development ?